packages/terra-form-checkbox/src/Checkbox.module.scss

```diff
@@ -1,3 +1,9 @@
+@import '~terra-mixins/lib/Mixins';
```

Comment:
do we need to add mixins here?

Reply:
I think i see what you're doing, I think it makes more sense to add the mixins include to each  of the theme scss files because thats where the mixin is used.

Reply:
Update here https://github.com/cerner/terra-core/pull/2971/commits/1502a47693f807fab75f3bb20f9dc7048900d901.